### ISSUE TITLE
depth_align_embs is not saved or optimized

### ISSUE THREAD

**Issue by LIJUNCHENG001** [Robbyant/lingbot-vla#58]
According to the paper: we apply the learnable queries [Qt1, Qt2, Qt3] corresponding to three-view operational images.
However, in the code, `depth_align_embs` is defined as:

```python
self.depth_align_embs = nn.Parameter(torch.randn(...)).to(torch.bfloat16)
```

`Parameter.to()` returns a regular Tensor, so it is absent from `named_parameters()`, optimizer groups, and checkpoints. It is randomly
recreated during loading, making depth output seed-dependent.

I have checked the pretrained model weights, `depth_align_embs` is indeed not saved.

Is this a bug or an intended design?

**Comment by zjn20030811** [comment]
I can take this issue on. I will keep the depth alignment embeddings as a registered bfloat16 Parameter so the optimizer and checkpoints include them, and add a lightweight regression covering gradients and state-dict round trips.

**Comment by PushpakAg** [comment]
It seems like `depth_align_embs` isn't being saved due to how it's defined. I'd check if its absence from `named_parameters()` is causing any unexpected behavior during training or inference.
